packages/runtime/runtime-definitions/src/events.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export const eventThis = Symbol("EventThis");
```

Comment:
>Symbol [](http://example.com/codeflow?start=25&length=6)

why use a symbol? this will actually create an artifact that exists at runtime.

Reply:
I did that to prevent collisions, I figured the cost was small, but I'm not really opposed to just picking an extremely unlikely collision as well.

Reply:
Oh I think I can achieve the same thing with `declare const eventThis: unique symbol;`.